tests/CartTest.php

```diff
@@ -1453,4 +1453,34 @@ private function setConfigFormat($decimals, $decimalPoint, $thousandSeperator)
         $this->app['config']->set('cart.format.decimal_point', $decimalPoint);
         $this->app['config']->set('cart.format.thousand_separator', $thousandSeperator);
     }
+
+    /** @test */
+    public function it_can_store__mutiple_instances_of_the_cart_in_a_database()
+    {
+        $this->artisan('migrate', [
+            '--database' => 'testing',
+        ]);
+
+        Event::fake();
+
+        $cart = $this->getCart();
+
+        $cart->add(new BuyableProduct());
+
+        $cart->store($identifier = 123);
+
+        $serialized = serialize($cart->content());
+
+        $newInstance = $this->getCart();
+        $newInstance->instance($instanceName = 'someinstance');
+        $newInstance->add(new BuyableProduct());
+        $newInstance->store($newIdentifier = 456);
```

Comment:
Wouldn't it be better to use the same identifier here to make sure the instance does get checked correctly?